### DISCUSSION TITLE
How to insert a slide at a specific position?

### DISCUSSION THREAD

**Discussion by alexklimuk** [ShapeCrawler/ShapeCrawler#147] [Q&A] [answered]
The example has "Copy second slide from source to target". Can I do this by specifying the position to insert the slide? Is there such a method?

**Comment by AdamShakhabov** [comment, marked as answer]
No, the current version does not contain a method to insert slide in a certain position.

As a workaround, you can add, then change the position of the added slide via setter of `ISlide.Number` property.
